pkg/virt-config/config-map.go

```diff
@@ -196,6 +197,7 @@ func defaultClusterConfig() *v1.KubeVirtConfiguration {
 		SELinuxLauncherType:         DefaultSELinuxLauncherType,
 		SupportedGuestAgentVersions: supportedQEMUGuestAgentVersions,
 		OVMFPath:                    DefaultOVMFPath,
+		MemBalloonPeriod:            DefaultMemBallonPeriod,
```

Comment:
I think this should be called `MemBalloonStatsPeriod` or something like that, because the period doesn't apply to the balloon itself but just to stats collection.

Additionally, shouldn't there be a way to document the configuration knob? For example its intended use, its default value, and the acceptable range, including the fact that a value of zero turns off stats collection.

Reply:
I think it should. I will document this in user-guide repo.
We could even do better by declining changes to virt-config while the user is editing it.

Reply:
You're still using `MemBalloonPeriod` instead of `MemBalloonStatsPeriod` or something along those lines, but it's not clear whether you disagree with my suggestion or you simply missed it...

Reply:
I missed it(was too late for me).